internal/clients/team7/agents/TeamSevenBiker.go

```diff
@@ -393,10 +390,25 @@ func (biker *BaseTeamSevenBiker) CreateGoverenceMessage() objects.GovernanceMess
 }
 
 func (biker *BaseTeamSevenBiker) CreateForcesMessage() objects.ForcesMessage {
-	return objects.ForcesMessage{
-		BaseMessage: messaging.CreateMessage[objects.IBaseBiker](biker, biker.GetFellowBikers()),
-		AgentId:     biker.GetID(),
-		AgentForces: biker.GetForces(),
+	// Low conscientiousness => Unethical => More likely to lie about forces.
+	// Low conscientiousness => Dependable => Less likely to lie about forces.
+	randNum := rand.Float64()
+	if biker.personality.Conscientiousness < randNum {
```

Comment:
Any reason for using a random number here? It means that our agent will have a different behaviour each time right?

Reply:
Yeah I was trying to model the likelihood of lying at each iteration being inversely proportionate to conscientiousness. Open to a different implementation though